user/lxc/template.py

```diff
@@ -0,0 +1,46 @@
+pkgname = "lxc"
+pkgver = "5.0.1"
+pkgrel = 1
+build_style = "meson"
+configure_args = [
+    "-Dapparmor=false",
+    "-Dman=false",
+    "-Dinit-script=sysvinit"
+]
+hostmakedepends = [
+    "meson",
+    "pkgconf",
+    "git"
+]
+makedepends = [
+    "libcap-devel",
+    "libcap-devel-static",
+    #"libapparmor-devel",
+    "libseccomp-devel",
+    "libunwind-devel",
+    "libunwind-devel-static",
+    "libatomic-chimera-devel",
+    "libatomic-chimera-devel-static",
+    "openssl-devel",
+    "musl-devel-static",
+]
+depends = [
+    "xz",
+    "wget2"
+]
+pkgdesc = "Linux containers"
+maintainer = "tj <tjheeta@gmail.com>"
+license = "GPL-2.0-only AND LGPL-2.1-only"
+url = "https://linuxcontainers.org"
+source = f"{url}/downloads/{pkgname}/{pkgname}-{pkgver}.tar.gz"
+sha256 = "d8195423bb1e206f8521d24b6cde4789f043960c7cf065990a9cf741dcfd4222"
+file_modes = {"usr/libexec/lxc/lxc-user-nic": ("root", "root", 0o4755)}
+
+def post_extract(self):
+    # https://github.com/void-linux/void-packages/pull/38493/files
+    # workaround https://github.com/lxc/lxc/issues/4178
+    self.mkdir("/etc/default", parents = True)
```

Comment:
this is fixed in lxc 6.0 already and you should use the latest version

Reply:
Good catch. Will do.